dwave/plugins/sklearn/transformers.py

```diff
@@ -153,27 +153,6 @@ def correlation_cqm(
             https://1qbit.com/whitepaper/optimal-feature-selection-in-credit-scoring-classification-using-quantum-annealer
         """
 
-        X = np.atleast_2d(np.asarray(X))
```

Comment:
I think we should keep these checks in each of the `_cqm` functions - we do encourage users to call them directly.

Reply:
created a function `_check_params` to be called from `_cqm` methods for checking